families/track_and_trade/client/src/fish_form.js

```diff
@@ -0,0 +1,236 @@
+/**
+ * Copyright 2017 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ----------------------------------------------------------------------------
+ */
+
+
+const m = require('mithril')
+const {submitRecord} = require('./transactions')
+
+
+const authorizableProperties = [
+  ['location', 'Location'],
+  ['temp', 'Temperature'],
+  ['tilt', 'Tilt'],
+  ['shock', 'Shock']
+]
+
+const MultiSelect = {
+  view (vnode) {
+    let handleChange = vnode.attrs.onchange || (() => null)
+    let selected = vnode.attrs.selected
```

Comment:
Same with the other `let`s in this PR. The use of `let` implies to me that the variable will be reassigned, which none of these are (at least at the moment). `const` is preferable.